core/src/components/header/header-brand-symbol/header-brand-symbol.tsx

```diff
@@ -9,17 +9,14 @@ import { inheritAriaAttributes } from '../../../utils/utils';
 export class TdsHeaderBrandSymbol {
   @Element() host: HTMLElement;
 
-  /** The href for the logo link. */
-  @Prop() linkHref: string = 'https://www.scania.com';
-
   render() {
     const inheritedLinkProps = {
       ...inheritAriaAttributes(this.host),
     };
     return (
       <Host>
         <tds-header-item>
-          <a {...inheritedLinkProps} href={this.linkHref}></a>
+          <slot {...inheritedLinkProps}></slot>
```

Comment:
The inheritedLinkProps can be removed. The aria attributes should be put directly on the slotted element instead in this case.